server/save.go

```diff
@@ -1,83 +0,0 @@
-package main
-
-import (
-	"encoding/json"
-	"time"
-
-	"github.com/pkg/errors"
-)
-
-func (p *Plugin) retreiveData() error {
-	j, err := p.API.KVGet("analytics")
-	if err != nil {
-		return errors.Wrap(err, "failed to get analytics from kv")
-	}
-	p.currentAnalytic = NewAnalytic()
-	if err := json.Unmarshal(j, p.currentAnalytic); err != nil {
-		p.API.LogError("failed to unmarshal analytics from kv use new one", "err", err.Error())
-		p.currentAnalytic = NewAnalytic()
-	}
-	return nil
-}
-
-func (p *Plugin) startCronSaver(poison chan bool) {
-	go func() {
-		for {
-			select {
-			case <-time.After(1 + time.Minute):
-				p.save()
-			case <-poison:
-				p.save()
-				return
-			}
-		}
-	}()
-}
-
-func (p *Plugin) save() {
```

Comment:
Maybe rename this to `saveCurrentAnalytic`?

Reply:
:heavy_check_mark: